client/my-sites/site-settings/form-general.jsx

```diff
@@ -442,6 +444,46 @@ const FormGeneral = React.createClass( {
 		);
 	},
 
+	helloVoteOption() {
```

Comment:
Do we intend to clean it up after? Some explanation would help, since I had trouble following what that does